modules/configuring-the-training-operator-permissions-without-kueue.adoc

```diff
@@ -0,0 +1,158 @@
+
+If you use Kueue to run PyTorch jobs, you can skip this section. 
+
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+ifndef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/using-data-science-projects_projects#creating-a-data-science-project_projects[Creating a data science project].
+endif::[]
+ifdef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#creating-a-data-science-project_projects[Creating a data science project].
+endif::[]
+
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+* You have access to a model.
+* You have access to data to train the model.
+
+.Procedure
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<admin_username>__ -p __<password>__
+----
+
+. Create a cluster role with the required access permissions, as follows:
+.. Create a YAML file named `pytorchjob_role.yaml`.
+.. Add the following `ClusterRole` object definition:
++
+[source]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pytorchjob-role
+rules:
+  - verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    apiGroups:
+      - kubeflow.org
+    resources:
+      - pytorchjobs
+  - verbs:
+      - get
+    apiGroups:
+      - kubeflow.org
+    resources:
+      - pytorchjobs/status
+
+----
+.. Save your changes in the `pytorchjob_role.yaml` file.
+.. Apply the configuration to create the `pytorchjob-role` object, as follows:
++
```

Comment:
general comment - seems like you could remove many (not all)  of these instances of ",as follows"

Reply:
Removed it from the substeps but kept it in the main steps :)